tests/e2e/scale/test_pvc_creation_deletion_scale.py

```diff
@@ -20,6 +18,56 @@ class TestPVCCreationDeletionScale(E2ETest):
     """
     Base class for PVC scale creation and deletion
     """
+    def code_inside_pvc_test(self, msg):
+        # Code that will be run inside the various pvc creation and deletion
+        # tests goes here
+        log.info(msg)
+
+    @pytest.fixture()
+    def pvc_scale_teardown(self, request):
+        def pvc_scale_deletion():
+            log.info("tearing down pvc creation scale test")
+            # Delete PVC
+            helpers.delete_objs(self.pvc_objs)
+
+            # Get PVC deletion time
+            pvc_deletion_time = helpers.measure_pv_deletion_time_bulk(
+                interface=self.interface, pv_name_list=self.pv_nm_list
+            )
+
+            helpers.write_csv_data(
+                pvc_deletion_time,
+                f"{self.log_path}-deletion-time.csv",
+                "Delete"
+            )
+        request.addfinalizer(pvc_scale_deletion)
+        return pvc_scale_deletion
+
+    @pytest.fixture()
+    def pvc_4_type_teardown(self, request):
```

Comment:
doc string needed and should get rid of '4_' in function name, what does it mean?

Reply:
The 4 is the number of combinations of pvc_objects and access modes tested.  There are four different combinations tested here.  These combinations are:

fs pvc, read write once
fs pvc, read write many
rbd pvc, read write once
rbd pvc, read write execute access

The 4 is in throughout the test_pvc_creation_deletion module (it was not my idea to use a 4 here).  I can change these
references to something like 'combinations' if that is preferable.